# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/thebes/gfxPlatformGtk.cpp
# Commit: 31fba4d1fe1d
# Full Hash: 31fba4d1fe1d16c5e0e4172bc78a8e26827849aa
# Author: Martin Stransky <stransky@redhat.com>
# Date: 2020-06-13 21:41:37
# Regressor Bug: 1629788
# File Overlap Count: 1
# Description:
#   Bug 1629788 [Wayland] Add widget.wayland-dmabuf-video-textures.enabled preference, r=jhorak
#   
#   - widget.wayland-dmabuf-video-textures.enabled controls dmabuf texture backend for decoded video frames.
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D78293
# ==============================================================================

diff -r 9d1e4da287a6 -r 31fba4d1fe1d gfx/thebes/gfxPlatformGtk.cpp
--- a/gfx/thebes/gfxPlatformGtk.cpp	Sat Jun 13 10:35:40 2020 +0000
+++ b/gfx/thebes/gfxPlatformGtk.cpp	Sat Jun 13 10:35:32 2020 +0000
@@ -725,6 +725,9 @@
 bool gfxPlatformGtk::UseWaylandDMABufTextures() {
   return IsWaylandDisplay() && nsWaylandDisplay::IsDMABufTexturesEnabled();
 }
+bool gfxPlatformGtk::UseWaylandDMABufVideoTextures() {
+  return IsWaylandDisplay() && nsWaylandDisplay::IsDMABufVideoTexturesEnabled();
+}
 bool gfxPlatformGtk::UseWaylandDMABufWebGL() {
   return IsWaylandDisplay() && nsWaylandDisplay::IsDMABufWebGLEnabled();
 }